compose/docker-compose.yml

```diff
@@ -42,9 +42,12 @@ services:
       test: curl --fail -I http://localhost:5687/openapi.json || exit 1
       interval: 1m
       start_period: 20s
+    ports:
+      - "5687:5687"  # Expose the main API port
```

Comment:
Does the REST API port need to be exposed. its already set to be exposed locally in `prod.yml`

Reply:
Agreed, removed